src/cryptography/utils.py

```diff
@@ -119,6 +119,9 @@ def __getattr__(self, attr):
     def __setattr__(self, attr, value):
         setattr(self._module, attr, value)
 
+    def __delattr__(self, attr):
+        delattr(self._module, attr)
```

Comment:
This should raise a warning if the attribute is deprecated, since deleting it won't work shortly.